src/__mocks__/react-native.js

```diff
@@ -4,7 +4,7 @@
 module.exports = {
   NativeModules: {
     RNSnowplowTracker: {
-      createTracker: (n,cc) => Promise.resolve(true)
-    }
-  }
+      createTracker: (n, cc) => Promise.resolve(true),
```

Comment:
based on the check warnings, it seems we can remove the `eslint-disable` comments in this file